### ISSUE TITLE
Save/Fork/Update

### ISSUE THREAD

**Issue by MasseGuillaume** [MasseGuillaume/ScalaKata#47]

**Comment by MasseGuillaume** [comment]
similar to [jsfiddle](http://jsfiddle.net)/[gist](http://gist.github.com)

**Comment by megri** [comment]
Would this mean that ScalaKata would be a publicly accessible/updatable example site? In that case, I'm all for it! I'm curious how to best manage spammers/vandals though.

**Comment by MasseGuillaume** [comment]
yep that's a great idea! I tough it could work like a gist site where people just use to share pieces of code. But having an executable wiki would make more sense. Wiki has a good structure to avoid vandalism.

**Comment by megri** [comment]
That's a cool idea. I've been playing around with it for some time (when it was also called CodeBrew) and it's definitely a useful platform. I like the gist-idea too, but it looks more like a wiki at the moment ;)
